src/Android/Avalonia.Android/AvaloniaActivity.cs

```diff
@@ -1,27 +1,50 @@
-
 using Android.App;
 using Android.OS;
 using Android.Views;
 
+using Avalonia.Rendering;
+
 namespace Avalonia.Android
 {
     public abstract class AvaloniaActivity : Activity
     {
-        
         internal AvaloniaView View;
         object _content;
 
         protected override void OnCreate(Bundle savedInstanceState)
         {
-            RequestWindowFeature(WindowFeatures.NoTitle);
             View = new AvaloniaView(this);
-            if(_content != null)
+            if (_content != null)
                 View.Content = _content;
             SetContentView(View);
             TakeKeyEvents(true);
             base.OnCreate(savedInstanceState);
         }
 
+        protected override void OnResume()
```

Comment:
That will require an Avalonia-based Activity which prevents AvaloniaView from being used for embedding scenarios

Reply:
I updated the PR to replace this with manual `Tick` subscription count. However, I noticed it isn't enough to stop the timer as the animation clock ([`Avalonia.Animation.RenderLoopClock`](https://github.com/AvaloniaUI/Avalonia/blob/6ff478bdb131b9643d3d3bb17caf50c5bcba3181/src/Avalonia.Visuals/Animation/RenderLoopClock.cs)) never stops, so the render loop doesn't stop, i.e. `_items.Count` never reaches 0 here:

https://github.com/AvaloniaUI/Avalonia/blob/6ff478bdb131b9643d3d3bb17caf50c5bcba3181/src/Avalonia.Visuals/Rendering/RenderLoop.cs#L87-L90

Is this a bug? Or am I missing something?

Reply:
It's a known issue. For now I'd suggest to manually cast IRenderTimer to a concrete type and manually notify it about the subscription count